### PR TITLE
Fix linting

### PR DESCRIPTION
**Pull request by eskaur**

### CODE REVIEW

File: .flake8-packaged-files [hunk 1]
```diff
@@ -10,6 +10,12 @@ ignore =
...
-    D402, # First line should not be the function’s “signature”
 show-source = True
+
+### Ignore docstring complaints in data models
+per-file-ignores =
+    settings.py:D101,D102,D106,D107
+    settings_2d.py:D101,D102,D106,D107
+    camera_state.py:D101,D102,D106,D107
+    camera_info.py:D101,D102,D106,D107
+    frame_info.py:D101,D102,D106,D107
```
Comment (trym-b) [inline .flake8-packaged-files:21]: You probably want to add a comment about what kind of errors D101, D102, D106 and D107 really are. Makes it easier to fix them in the right order later

File: samples/sample_capture_assistant.py [hunk 2]
```diff
@@ -10,7 +10,7 @@ def _main():
 
     suggest_settings_parameters = SuggestSettingsParameters(
         max_capture_time=datetime.timedelta(milliseconds=1200),
-        ambient_light_frequency=SuggestSettingsParameters.AmbientLightFrequency.none,
+        ambient_light_frequency="none",
```
Comment (trym-b) [inline samples/sample_capture_assistant.py:13]: Why was this changed?

**Review by trym-b — COMMENTED** [review]
Added some minor comments, looks good

File: samples/sample_capture_assistant.py (hunk 2 above)
Comment (eskaur) [reply samples/sample_capture_assistant.py:13]: See the relevant commit message:
```
I believe this is better, as it makes using the CA seem easier.  Strings
are the common way of inputting such arguments in Python anyway.
```

I don't feel strongly about this though, I have just found myself being annoyed at typing out a lot of stuff lately when wanting to use the CA in python, and just realized that they are strings now. 
Feel free to disagree, or say that it doesn't belong in this PR :)

File: .flake8-packaged-files (hunk 1 above)
Comment (eskaur) [reply .flake8-packaged-files:21]: Good idea, I'll do that.

Comment (eskaur) [reply .flake8-packaged-files:25]: Done

File: samples/sample_capture_assistant.py (hunk 2 above)
Comment (trym-b) [reply samples/sample_capture_assistant.py:13]: It might be easier to discover the other values ambient light frequenct could be if there with the previous solution. If a user just wants to play around with this sample and wants to change the frequency to something else, the user`s IDE could potentially help them out.

File: .flake8-packaged-files (hunk 1 above)
Comment (trym-b) [reply .flake8-packaged-files:25]: Nice

File: samples/sample_capture_assistant.py (hunk 2 above)
Comment (eskaur) [reply samples/sample_capture_assistant.py:13]: Good point. I can revert this.

Comment (eskaur) [reply samples/sample_capture_assistant.py]: Done.

**Comment by eskaur** [comment]
Ready for next round, @trym-b

**Review by trym-b — APPROVED** [review]
Looks good to me, I am happy that this branch now passes the CI

**Comment by eskaur** [comment]
Oh no, the ArchLinux build is set to always use the latest packages on AUR. I just upgraded AUR to 2.1 now, so any build targeting 2.0 will fail.
